helm/chart-operator-chart/templates/deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Values.name }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: {{ .Values.name }}
+spec:
+  replicas: {{ .Values.replicas }}
+  revisionHistoryLimit: 3
```

Comment:
IMO this should be 1 because this project here is an operator and we should never run multiple instances of them.

Reply:
Agree we should run 1 replica and align with the charts of our existing operators.

Reply:
Makes sense!